src/vs/editor/common/services/languagesRegistry.ts

```diff
@@ -155,10 +155,7 @@ export class LanguagesRegistry extends Disposable {
 		if (Array.isArray(lang.extensions)) {
 			for (let extension of lang.extensions) {
 				mime.registerTextMime({ id: langId, mime: primaryMime, extension: extension }, this._warnOnOverwrite);
-				// {{SQL CARBON EDIT}}
```

Comment:
This is a revert of #779, removing the language declaration from mssql is a cleaner fix from a merge conflict point of view. I tested and intellisense still works.